src/styles/FarmSearch.css

```diff
@@ -0,0 +1,98 @@
+* {
+  box-sizing: border-box;
+}
+
+.farmsearch-container {
+  margin: 5%;
+  height: 100%;
+}
+
+.farmsearch-header {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 3.75%;
+  font-weight: bold;
+  font-size: 36px;
+  line-height: 100%;
+  letter-spacing: 1px;
+}
+
+.farmsearch-box {
+  position: relative;
+  width: 440px;
+  height: 32px;
+}
+.farmsearch-box input[type='text'] {
+  border: none;
+  width: 100%;
+  height: 100%;
+  border-radius: 16px;
+  padding-left: 17px;
+  color: #b8b8b8;
+  font-size: 15px;
+  vertical-align: middle;
+}
+
+.farmsearch-box input[type='image'] {
+  position: absolute;
+  right: 0;
+  top: 0;
+  margin-top: 15px;
+  margin-right: 10px;
+}
+
+.farmsearch-filterbox {
+  width: 17%;
+  height: 40px;
+  background-color: #fafafa;
+  border: 1px solid rgba(51, 51, 51, 0.5);
+  border-radius: 5px;
+  color: rgba(51, 51, 51, 0.5);
+  font-size: 14px;
+  text-align: left;
+  padding-left: 15px;
+}
+
+.farmsearch-gridcontainer {
+  display: grid;
+  grid-template-columns: 27.77% 27.77% 27.77%;
```

Comment:
Tweak the css grid code to make it look better? Also maybe add a fixed space between the "GAP Certification Stage" and "Sort by" buttons.

Reply:
done